src/openapi.yaml

```diff
@@ -1122,32 +1116,26 @@ paths:
         $ref: '#/components/requestBodies/EnkelvoudigInformatieObjectData'
     delete:
       operationId: enkelvoudiginformatieobject_delete
-      summary: 'Verwijdert een ENKELVOUDIG INFORMATIEOBJECT, samen met alle gerelateerde
+      description: 'Delete the canonical EnkelvoudigInformatieObject, which cascade
```

Comment:
did you install the latest requirements versions? this looks like a downgrade of drf-yasg